src/main/docker/Dockerfile

```diff
@@ -36,7 +36,7 @@ RUN if [ "${cantaloupe.version}" = "dev" ] ; then \
       mvn -q install:install-file -Dfile="/tmp/${JAI_IMAGEIO_JAR}" -DgroupId="javax.media" \
         -DartifactId="jai_imageio" -Dversion="$JAI_IMAGEIO_VERSION" -Dpackaging="jar" && \
       \
-      mvn -DskipTests=true -q clean package && \
+      mvn -DskipTests=true -Djar.phase=package -q clean package && \
       mv target/cantaloupe-?.?-SNAPSHOT.zip "/build/Cantaloupe-${cantaloupe.version}.zip" ; \
```

Comment:
Hmm, I'd forgotten we didn't translate the devBuild Travis builds to GitHub Actions... we only seem to be building with and without Kakadu at the moment. And, I did contemplate removing the dev build now that the upstream Cantaloupe has released v5. What are your thoughts on that? Do we need a dev build for the container still?

Reply:
This decision was made before my time so I may be missing something, but it may still be nice to have in case e.g. we discover a bugfix we need that hasn't been released yet, so I'd say if it's easy get it working again then why not keep it in?

Reply:
I'm not convinced of its usefulness now that Cantaloupe is officially on v5, but it would probably take more time to strip it out than we currently have so I guess it's okay to let it ride for a bit longer. I pushed a fix for the broken devBuild and test issues.